lib/trade_tariff_frontend.rb

```diff
@@ -93,10 +93,6 @@ def regulations_enabled?
     ENV.fetch('HIDE_REGULATIONS') != 'true'
   end
 
-  def download_pdf_enabled?
-    ENV.fetch('DOWNLOAD_PDF_ENABLED', 'false') == 'true'
```

Comment:
Will you remove it from the PaaS as well?

Reply:
As in the env vars? Yeah , absolutely :)